chain.schema.json

```diff
@@ -83,28 +84,8 @@
       },
       {
         "name": "v15",
-        "tag" : "v15.1.0",
-        "height": 8732500,
-        "recommended_version": "v15.1.0",
-        "compatible_versions": [
-          "v15.1.0",
-          "v15.0.0"
-        ],
-        "cosmos_sdk_version": "0.45",
-        "consensus": {
-          "type": "tendermint",
-          "version": "0.34"
-        },
-        "cosmwasm_version": "0.30",
-        "cosmwasm_enabled": true,
-        "ibc_go_version": "4.3.0",
-        "ics_enabled": [
-          "ics20-1"
-        ],
-        "binaries": {
```

Comment:
can we please update the binaries here is the version object and keep all the other properties (sdk version, cw version, consensus, ibc version, etc.)?

Reply:
pinging @niccoloraspa for vis